package.json

```diff
@@ -173,10 +173,12 @@
     "iti": "0.6.0",
     "jest": "29.7.0",
     "jest-environment-jsdom": "29.7.0",
+    "jest-fixed-jsdom": "0.0.9",
     "jquery": "3.7.1",
     "lint-staged": "10.5.4",
     "mini-css-extract-plugin": "2.3.0",
     "moment": "2.29.4",
+    "msw": "1.3.2",
```

Comment:
I'm using an older version of `msw`, because the newer ones are not compatible with our TS version